addons/gametime/gametime.lua

```diff
@@ -247,6 +247,14 @@ config.register(settings, function()
     end
     gt.gtd:text('${day|} ${MoonPhase|Unknown} (${MoonPct|-}%); ${WeekReport|}')
 
+	if tonumber(settings.numdays) ~= nil and settings.numdays > 0 and settings.numdays < 9 then
```

Comment:
The `tonumber(settings.numdays) ~= nil` can be removed, it should always be `true`. If it's anything else it would be complaining much earlier already (when parsing the user settings).